easy_my_coop_website_taxshelter/views/easy_my_coop_website_taxshelter_templates.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright 2015-2016 Odoo S.A.
+     Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
```

Comment:
remove non contributor of this module

Reply:
I put these copyrights because I copied the template `portal_my_home_menu_tax_shelter` from the website_portal_v10 package.